adafruit_il0398.py

```diff
@@ -23,13 +23,15 @@
 
 """
 
-# Support both 8.x.x and 9.x.x. Change when 8.x.x is discontinued as a stable release.
+from epaperdisplay import EPaperDisplay
+
 try:
+    import typing
+
     from fourwire import FourWire
-    from epaperdisplay import EPaperDisplay
 except ImportError:
-    from displayio import FourWire
-    from displayio import EPaperDisplay
+    pass
```

Comment:
If the `import typing` fails, then `FourWire` won't be imported. I don't think this is what you meant.

Reply:
@dhalbert that is the intention. The only usage of `FourWire` is here: https://github.com/adafruit/Adafruit_CircuitPython_IL0398/blob/main/adafruit_il0398.py#L60 as a type annotation. If `typing` fails we know its a MCU context and this allows us to avoid the memory overhead of importing `FourWire` that won't be used at runtime.

Reply:
Thanks -- I realized this when I was reviewing another one.